### DISCUSSION TITLE
Add percentage setting for Base Layer height

### DISCUSSION THREAD

**Discussion by Bumblebiber** [supermerill/SuperSlicer#1762] [Community contribution]
Hey guys,

I use many different sized nozzles. It's annoying to create and update print profiles for each of the nozzles separately. 

Wouldn't it be great, if I could use percentage values for Base Layer height? So I just needed like three profiles: one for fast and rough prints with 75 to 80% of the nozzle size, one intermediate with 40 to 50% and one fine setting with around 10 to 20% of the nozzle diameter. It's considerable to round the calculated values to one that fits a Z full step.

With my formerly used PrusaSlicer it looks like this:
![grafik](https://user-images.githubusercontent.com/78353703/139642687-33a8b1fa-622c-4fcb-b105-f65d3c3fa1e5.png)

I would highly appreciate such a feature, what do you think?

**Comment by supermerill** [comment]
already submitted in #219
